### DISCUSSION TITLE
Need clarity regarding support policy

### DISCUSSION THREAD

**Discussion by shazib-summar** [vectordotdev/vector#20631] [Q&A] [answered]
Hello. The release cadence for Vector at https://github.com/vectordotdev/vector/blob/master/RELEASES.md#cadence says

> Stable channel
> - Every 6 weeks
> - Release patch fixes as needed to fix high-priority bugs and regressions from the last major or minor release
> - Release daily builds representing the latest state of Vector for feedback

Im confused by what "from the last major or minor release" means in the second point. Does it mean
- bug fixes are published to the last minor release only
- bug fixes are published to all the current minor release and the latest minor release from the last major release

Thanks in advance...

**Comment by jszwedko** [comment, marked as answer]
Hi @shazib-summar !

Because Vector is still < 1.0.0, it effectively means that bug fixes are only published to the last minor release (0.X.Y).

**Reply by shazib-summar** [reply]
Hey @jszwedko, thanks a lot for the help. A follow up question; would the support policy be the same once 1.0.0 is released (and afterwards)? Or is this not decided yet.

Thanks again

**Reply by jszwedko** [reply]
It is intended to be the same, but I think we will likely re-evaluate once it is 1.0.0 since we haven't revisited this policy in a while.

**Reply by shazib-summar** [reply]
@jszwedko can I create a PR to update the RELEASES.md file? The explanation you provided is much clearer than what https://github.com/vectordotdev/vector/blob/master/RELEASES.md says

**Reply by jszwedko** [reply]
@shazib-summar sure! That would be welcome.
